leapp/utils/audit/__init__.py

```diff
@@ -651,10 +651,15 @@ def store_actor_metadata(actor_definition, phase):
         'consumes': sorted(model.__name__ for model in _metadata.get('consumes', ())),
         'produces': sorted(model.__name__ for model in _metadata.get('produces', ())),
         'tags': sorted(tag.__name__ for tag in _metadata.get('tags', ())),
+        'config_schemas': [field.serialize() for field in _metadata.get('config_schemas', ())],
     })
     _metadata['phase'] = phase
 
-    actor_metadata_fields = ('class_name', 'name', 'description', 'phase', 'tags', 'consumes', 'produces', 'path')
+    # TODO(dkubek): Is configuration metadata? Does it make sense to include it here?
+    _metadata['config'] = config
```

Comment:
I have removed it from the metadata. I have decided to store it in a separate table, so that it is linked only to 'execution'. I was considering multiple possibilities, but in the end this was one that made the most sense to me.

Reply:
Good job! Closing the conversation to keep track of what still needs work/discussion.